### ISSUE TITLE
incorrect storage reporting with in-memory store

### ISSUE THREAD

**Issue by RaduBerinde** [cockroachdb/cockroach#35474]
Started a single node with:
```
./cockroach start --insecure --logtostderr --store=type=mem,size=4GiB
```

The UI shows 512MiB capacity. And it shows 0 used, even though I have data in there (live bytes graph shows about 1GB).

![image](https://user-images.githubusercontent.com/16544120/53912206-6f321980-4026-11e9-99fe-7146f875fcc2.png)

**Comment by RaduBerinde** [comment]
This might be more than a presentation problem if the same values are internally used to make decisions about when to run GC.

**Comment by knz** [comment]
this has been fixed in 21.1
